<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatsPlayerInfoWidget</class>
 <widget class="QWidget" name="StatsPlayerInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>60</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: transparent;
    border: none;</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>295</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:none;
     background: transparent;
     background-image:none;
     border:none;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_playerColor">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>5</y>
      <width>295</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;
      background: transparent;
      background-image:none;
      border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../resources/Resources.qrc">:/images/player_blue.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_username">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>5</y>
      <width>121</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Halo Dek</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(96, 56, 19);
      border-image:none;
      background: transparent;
      background-image:none;
      border:none;</string>
    </property>
    <property name="text">
     <string>username</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_scoreBg">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>5</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:none;
      background: transparent;
      background-image:none;
      border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../../resources/Resources.qrc">:/images/button_small.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_score">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>5</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Halo Dek</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(194, 65, 0);</string>
    </property>
    <property name="text">
     <string>100</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
